tests/scripts/simple_anisotropic_tiff_cubing.sh

```diff
@@ -10,6 +10,7 @@ docker run \
   --batch_size 8 \
   --layer_name color \
   --max 8 \
+  --scale "11.24,11.24,25"\
```

Comment:
Do you know why the CI passed even before this change? Isn't this called in the CI?

Reply:
Sry I don't know.